apps/notifier/src/pull-listener.ts

```diff
@@ -15,14 +21,81 @@ export default async function startPullListener(): Promise<void> {
     const subscription = await pubsubClient.getSubscription(pubsubSettings.topicName, pubsubSettings.subscriptionName)
     logger.info('listening messages from topics')
 
+    sgMail.setApiKey(SENDGRID_APY_KEY);
+
     subscription.on('message', async (message: Message) => {
       const { attributes } = message
+      const { topic } = attributes;
+      const dataJson = JSON.parse(message.data.toString());
+
+      const pathToAttachment = `${__dirname}/src/assets/views/hola.html`;
+      //const attachment = fs.readFileSync(pathToAttachment,'utf8');
+      console.log("antes del template")
+      nunjucks.configure('views',{autoescape: true});
+      console.log("en medio del template") 
+      // D:\Escritorio\Master\Fundamentos de cloud\bike4life\apps\notifier\src\views\base.html
+      // D:\Escritorio\Master\Fundamentos de cloud\bike4life\apps\notifier\src\pull-listener.ts
+
+      const Nuntemplate = nunjucks.render('/views\/base.html',dataJson);
+      console.log("despues del template")
+    
+      
+
+      if(topic === "UserCorrectlyCreated") {
+        console.log('hola');
+        const msg = {
+          to: dataJson.user_email,
+          from: TO_REPLY_TO_EMAIL_SENDGRID,
+          reply_to: TO_REPLY_TO_EMAIL_SENDGRID,
+          subject: 'User created correctly',
+          content:[
+            {
+              type: 'text/html',
+              value: Nuntemplate
+            }
+          ],
+          text: "hola",
+          /*attachments: [
+            {
+              content: attachment,
+              filename: "attachment.html",
+              type: "text/html",
+              disposition: "attachment"
+            }
+          ]*/
+        }
+      } else if(topic === "RouteOptimizedNotification"){
```

Comment:
This logic should be implemented in other place. The pull listener file's code should just hear for messages and call a function that could be name `handleMessage(type, payload)` that will handle all the required steps please.